src/labkey/Domain.ts

```diff
@@ -408,6 +408,58 @@ export function save(config: SaveDomainOptions): XMLHttpRequest {
     });
 }
 
+export interface ValidateDomainNameExpressionOptions extends CreateDomainOptions {
+    includeNamePreview: boolean
+}
+/**
+ * Validate name expression(s) for domain design.
+ */
+export function validateNameExpressions(config: ValidateDomainNameExpressionOptions): XMLHttpRequest {
+
+    let options: ValidateDomainNameExpressionOptions = arguments.length > 1 ? {
+        success: arguments[0],
+        failure: arguments[1],
+        domainDesign: arguments[2],
+        options: arguments[3],
+        kind: arguments[4],
+        includeNamePreview: arguments[5],
+    } : config;
+
+    return request({
+        url: buildURL('property', 'validateNameExpressions.api', options.containerPath),
+        method: 'POST',
+        success: getCallbackWrapper(getOnSuccess(options), options.scope),
+        failure: getCallbackWrapper(getOnFailure(options), options.scope, true),
+        jsonData: {
+            domainDesign: options.domainDesign,
+            options: options.options,
+            kind: options.kind
+        }
+    });
+}
+
+export function getDomainNamePreviews(config: GetDomainDetailsOptions): XMLHttpRequest {
+
+    let options: GetDomainDetailsOptions = arguments.length > 1 ? {
+        containerPath: arguments[4],
+        failure: arguments[1],
+        queryName: arguments[3],
+        schemaName: arguments[2],
+        success: arguments[0]
+    } : config;
```

Comment:
Ok, really don't know how the order of these arguments is determined.  What am I missing?